js/modules/auto_orient_image.js

```diff
@@ -0,0 +1,45 @@
+const loadImage = require('blueimp-load-image');
+
+const DEFAULT_JPEG_QUALITY = 0.85;
+
+// File | Blob | URLString -> LoadImageOptions -> Promise<DataURLString>
+//
+// Documentation for `options` (`LoadImageOptions`):
+// https://github.com/blueimp/JavaScript-Load-Image/tree/v2.18.0#options
+exports.autoOrientImage = (fileOrBlobOrURL, options = {}) => {
```

Comment:
Regarding `options = {}`, I've noticed that if someone passes `null` for the options hash, the default doesn't take effect, and you can end up with null-ref errors. I generally go for `options = options || {}` to be sure.

Reply:
Good point! I tried but it violates our `no-param-reassign` ESLint rule. I’d say explicitly passing in `null` for an optional argument is unlikely to be accidental. In terms of addressing general null-safety, we can do that later on with better static analysis.